configuration/cdap-site.xml

```diff
@@ -578,6 +571,67 @@
   </property>
   -->
 
+  <!-- Metadata Configuration -->
+
+  <property>
+    <name>metadata.max.allowed.chars</name>
+    <value>50</value>
+    <description>
+      Maximum number of characters for metadata keys, values, and tags
+    </description>
+  </property>
+
+  <property>
+    <name>metadata.service.bind.address</name>
+    <value>0.0.0.0</value>
+    <description>
+      Default inet address for binding metadata HTTP service
+    </description>
+  </property>
+
+  <property>
+    <name>metadata.service.exec.threads</name>
+    <value>${http.service.exec.threads}</value>
+    <description>
+      Number of Netty server executor threads for metadata HTTP service
+    </description>
+  </property>
+
+  <property>
+    <name>metadata.service.worker.threads</name>
+    <value>${http.service.worker.threads}</value>
+    <description>
+      Number of Netty server IO worker threads for metadata HTTP service
+    </description>
+  </property>
+
+  <property>
+    <name>metadata.updates.kafka.broker.list</name>
+    <value>127.0.0.1:${kafka.bind.port}</value>
+    <description>
+      Kafka broker list to which metadata update notifications are published
+    </description>
+  </property>
+
+  <property>
+    <name>metadata.updates.kafka.topic</name>
+    <value>cdap-metadata-updates</value>
+    <description>
+      Kafka topic to which metadata update notifications are published
+    </description>
+  </property>
+
+  <property>
+    <name>metadata.updates.publish.enabled</name>
+    <value>true</value>
```

Comment:
Ahh, thanks... I must have gotten the wrong value from one of the PRs and missed it being changed.